### ISSUE TITLE
Center justifying is problematic for readability in Guided Assistant

### ISSUE THREAD

**Issue by KellyeDC-zz** [LSCOIT/LegalNavigator#924]
Center justifying is problematic for readability in Guided Assistant

Issue submitted by Jeannie Sato (AK)

**Comment by breeglej** [comment]
@KellyeDC - Because this appears to be a design change, I'd like to save it for the advisory board and put it on the first agenda.

**Comment by grawdon** [comment]
@KellyeDC Agreed. Fell Swoop must have done this for a reason. I'd like some expert input on this. Perhaps I could reach out to Maria Mindlin at Transcend and see what they advise.
